packages/checkout/widgets-lib/src/components/RawImage/RawImage.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, useConvertSxToEmotionStyles } from '@biom3/react';
+import { imageStyles, rawImageStyles } from './RawImageStyles';
+
+export interface RawImageProps {
```

Comment:
not a blocker, to keep the type proper, you could import `BoxProps` and  do `sx: BoxProps['sx']`

Reply:
Nice input, patched this.